src/xtt.c

```diff
@@ -247,27 +248,102 @@ enftun_xtt_handshake(const char *server_ip,
     }
 }
 
-static int read_tls_root_cert(TSS2_TCTI_CONTEXT *tcti_context, const char* ca_cert_file)
+static int read_tls_root_cert(unsigned char* tls_root_cert, const char* ca_cert_file, uint16_t tls_len)
```

Comment:
Since this function is now just literally wrapping `xtt_save_to_file`, let's remove it entirely.  Just  call `xtt_save_to_file` directly instead.